Platform/CommonBoardPkg/Include/Library/BoardSupportLib.h

```diff
@@ -21,4 +21,38 @@ FillBootOptionListFromCfgData (
   IN OUT   OS_BOOT_OPTION_LIST   *OsBootOptionList
 );
 
+
+/**
+  Set the platform name with CFGDATA info
+
+ **/
+VOID
+EFIAPI
+PlatformNameInit (
+  VOID
+);
+
+/**
+  Load the configuration data blob from SPI flash into destination buffer.
+  It supports the sources: PDR, BIOS for external Cfgdata.
+
+  @param[in]    Dst        Destination address to load configuration data blob.
+  @param[in]    Src        Source address to load configuration data blob.
+  @param[in]    Len        The destination address buffer size.
+
+  @retval  EFI_SUCCESS             Configuration data blob was loaded successfully.
+  @retval  EFI_NOT_FOUND           Configuration data blob cannot be found.
+  @retval  EFI_OUT_OF_RESOURCES    Destination buffer is too small to hold the
+                                   configuration data blob.
+  @retval  Others                  Failed to load configuration data blob.
+
+**/
+EFI_STATUS
+EFIAPI
+SpiLoadExternalConfigData (
```

Comment:
Why do we call SpiLoadExternalConfigData ?  Do we intend to load from somewhere else ?

Reply:
yes. External config data could come from other places, e.g. UFS or NVMe. The platform code could decide where to load it, and potentially platform code could have its own impl (e.g. we don't need move LoadExternalConfigData() from ConfigDataLib to BoardSupportLib, and it is a compatible change).

Reply:
ok